posthog/cdp/templates/make/template_make.py

```diff
@@ -0,0 +1,58 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+
+
+template: HogFunctionTemplate = HogFunctionTemplate(
+    status="free",
```

Comment:
Why free?

Reply:
Because our Zapier integration is free. Should we make it paid only?
![2024-11-19 at 15 36 02](https://github.com/user-attachments/assets/908843b9-473e-4b9e-b889-f41cf29ac19f)

Reply:
No the zapier integration is only free because it was before we setup this whole thing. Nothing should be free other than Slack and Zapier currently